check/check_test.go

```diff
@@ -145,21 +145,75 @@ var _ = Describe("Check", func() {
 
 						It("outputs a version containing the current time and supplied version", func() {
 							Expect(response).To(HaveLen(2))
-							Expect(response[0].Time.Unix()).To(BeNumerically("~", prev.Unix(), 1))
+							Expect(response[0].Time.Unix()).To(Equal(prev.Unix()))
 							Expect(response[1].Time.Unix()).To(BeNumerically("~", time.Now().Unix(), 1))
 						})
 					})
+
+					Context("from a predictable implementation", func() {
```

Comment:
I can see the the difference between tests under this context and previous is `prev` with `Truncate()`. Do we need to have both or could we just keep the one with `Truncate`?

Reply:
The purpose of having these separated was to make sure that the new implementation behaves correctly when provided (1) a version from the _**new**_ implementation, which is predictable and (2) a version from the _**old**_ implementation, which is not predictable.

In the "predictable" test context, we can truncate the value `prev` since all versions are defined with minute-level precision. In the other, existing text contexts, we can't perform that same operation.